src/Modules/ModulesGraded.jl

```diff
@@ -1550,7 +1614,14 @@ julia> I = ideal(S, gens(S))
 ideal(x[1], x[2], x[3], x[4])
 
 julia> FI = free_resolution(I)
-Free resolution of I
+Free resolution of Graded submodule of Graded multivariate polynomial ring^1(Element of
+GrpAb: Z
+with components [0])
+1 -> x[1]*e[1]
```

Comment:
This is another instance where printing should be repaired, probably similar to the above.